lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -158,6 +158,16 @@ export class AuthorizationCodeClient extends BaseClient {
             parameterBuilder.addCodeVerifier(request.codeVerifier);
         }
 
+        if (this.config.clientCredentials.clientSecret) {
```

Comment:
Nit: Strict null checks before this API would prevent  this failing? I mean what happens if `config.clientCredentials` is null?

Reply:
Since there is a default to `clientCredentials` it should be fine. Please add a test case though.

Reply:
Added test